src/plugins/vis_builder/public/visualizations/vislib/common/create_vis.ts

```diff
@@ -17,5 +19,14 @@ export const createVis = async (
   vis.data.searchSource = await getSearchService().searchSource.create();
   vis.data.searchSource.setField('index', indexPattern);
 
+  const responseAggs = vis.data.aggs.getResponseAggs().filter((agg) => agg.enabled);
+  responseAggs.forEach((agg) => {
+    if (isDateHistogramBucketAggConfig(agg)) {
```

Comment:
Good catch. I know the date_histogram bucket agg wont construct the label correctly without this value, but i dont see any reason why it would be a problem for other aggs too. Also this makes me want to spend a lil more time on this change to understand how the agg timeranges work. will update it in a revision